package.json

```diff
@@ -102,6 +102,7 @@
   "dependencies": {
     "@babel/core": "^7.5.5",
     "@babel/register": "^7.5.5",
+    "@elastic/apm-rum": "^4.4.2",
```

Comment:
Should this be a `devDependency`? Or is the longer term goal to have this enabled for non-devs?

Reply:
Due to the way bundling works on the Kibana side for core and X-Pack plugins, Its better to have it as dep's and not distribute the code with Kibana builds. 

I am exploring different ways and also talking to Kibana devs, will post details how we can do it in a better way without affecting the users.